### PR TITLE
Minor patch to the permissions since bluetooth api is still

### PR DESCRIPTION
**Pull request by shusain**
considered experimental on Chrome/Linux implementation.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -163,6 +163,12 @@ function createWindow() {
   });
 
   app.commandLine.appendSwitch('disable-features', 'OutOfBlinkCors')
+  
+  if (process.platform === "linux"){
+    app.commandLine.appendSwitch("enable-experimental-web-platform-features", true);
+  } else {
+    app.commandLine.appendSwitch("enable-web-bluetooth", true);
```
Comment (mmosca) [inline js/main.js:170]: Does seting enable-web-bluetooth to true cause issues on linux?

Can we just add the enable-experimental-web-platform-features, if linux instead?

**Comment by stronnag** [comment]
Tested on Arch Linux x64 and SB Nano3 BLE device.
Works as expected.

**Review by stronnag — APPROVED** [review]

**Comment by stronnag** [comment]
Also tested with a CC2541 BLE device. This is recognised, but the configurator refuses to communicate with it. 
```
2024-08-21 @ 18:49:59 -- Connected to: BleTest00
2024-08-21 @ 18:50:01 -- BLE device type: CC2541 based
2024-08-21 @ 18:50:01 -- MSP connection successfully opened with ID: 255
2024-08-21 @ 18:50:11 -- No configuration received within 10 seconds, communication failed
2024-08-21 @ 18:50:11 -- MSP connection successfully closed
```
However the device works in mwp.
```
18:53:27.832463 BLE chipset CC2541, mtu 23 (unlikely to end well)
18:53:27.832854 Connected 34:14:B5:B9:50:4E BleTest00 (nopoll false)
18:53:27.833057 Serial 34:14:B5:B9:50:4E BleTest00 (7e) ready false
18:53:28.218653 MSP Error: MSP_CMDS_IDENT[100,0b] 
18:53:28.608726 Using MSP v2 0205
18:53:29.226866 Model name: "BENCHYMCTESTY"
18:53:29.778390 V2 mixer 8 1
```

Wouldn't be too concerned about this, as CC2541 devices are pretty crap (MTU of 23 bytes indeed).

**Comment by stronnag** [comment]
For completeness, tested on aarch64 as well:
* Usually fails to detect any BLE devices
* When it does, the configurator fails to connect to the FC
* Works without problem in mwp (so this is an Electron/INAV issue not OS/architecture).

**Pull request merged**